src/App/App.tsx

```diff
@@ -87,56 +105,93 @@ class App extends Component<Props, State> {
     return '--/--/----';
   }
 
+  handleComparisonClick = (dropdownValue: ComparisonCategory): void => {
+    let comparisonStats = comparisonData.find(datum => {
+      return datum.category === dropdownValue;
+    })
+    this.setState({ selectedComparison: comparisonStats })
+  }
+
+  blockUnintededRoutes = (): string => {
+    let list = comparisonData.reduce((comparisonCategory, cat) => {
+      comparisonCategory += '|' + cat.category
+      return comparisonCategory
+    }, '')
+    return list.slice(0)
+  }
+
   render() {
     return(
-      <section>
+      <>
         <header>
-          <h1 className='title'>C🦠C🦠</h1>
-          <h3 className='tagline'>Covid Comparisons</h3>
+          <NavLink to='/' className='header'> 
+            <h1 className='title'>C🦠C🦠</h1>
+            <h3 className='tagline'>Covid Comparisons</h3>
+          </NavLink>
         </header>
-        <section>
-          <h3 className='usa-overview'>
-            <section className='overview-title'>
-              USA Overview: &nbsp; &nbsp;
-            </section>
-            <section className='overview-stats'>
-              <span className='overview-numbers'>{ new Intl.NumberFormat('en-US').format(this.state.allUSAData.positive) }</span> cases &nbsp; &nbsp; &nbsp; &nbsp;
-              <span className='overview-numbers'>{ new Intl.NumberFormat('en-US').format(this.state.allUSAData.death) }</span> deaths &nbsp; &nbsp; &nbsp; &nbsp;
-              <span className='overview-numbers'>{ new Intl.NumberFormat('en-US').format(this.state.allUSAData.hospitalizedCurrently) }</span> current hospitalizations 
+        <Switch>
+          <Route exact path='/'>
+            <section>
+              <section>
+                <h3 className='usa-overview'>
+                  <section className='overview-title'>
+                    USA Overview: &nbsp; &nbsp;
+                  </section>
+                  <section className='overview-stats'>
+                    <span className='overview-numbers'>{ new Intl.NumberFormat('en-US').format(this.state.allUSAData.positive) }</span> cases &nbsp; &nbsp; &nbsp; &nbsp;
+                    <span className='overview-numbers'>{ new Intl.NumberFormat('en-US').format(this.state.allUSAData.death) }</span> deaths &nbsp; &nbsp; &nbsp; &nbsp;
+                    <span className='overview-numbers'>{ new Intl.NumberFormat('en-US').format(this.state.allUSAData.hospitalizedCurrently) }</span> current hospitalizations 
```

Comment:
I think we should break this formatting out into a function to get rid of repetitive code.